lib/cli.js

```diff
@@ -471,10 +471,24 @@ export default async function loadCli() { // eslint-disable-line complexity
 		});
 	}
 
-	api.on('run', plan => {
+	api.on('run', async plan => {
+		if (combined.onInternalEvent) {
+			await combined.onInternalEvent({
```

Comment:
There isn't much use in awaiting this, and I'd use optional chaining:

```suggestion
		combined.onInternalEvent?.({
```

Reply:
84dba1d7